ui/pom.xml

```diff
@@ -11,5 +11,63 @@
 
     <artifactId>ui</artifactId>
 
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <failOnMissingWebXml>false</failOnMissingWebXml>
 
-</project>
\ No newline at end of file
+    <ui.maven.plugin.version>1.6</ui.maven.plugin.version>
+    <node.version>v8.10.0</node.version>
```

Comment:
These are already defined in the parent pom. You could either use the parent's or delete the parents' if it makes more sense to have it at this module's level